bedrock/settings/base.py

```diff
@@ -1337,6 +1343,11 @@ def JINJA_CONFIG():
     'firefox_os_mwc_2015_preview': {
         'source_filenames': (
             'js/base/mozilla-modal.js',
+            'js/libs/jquery.waypoints.min.js',
+            'js/libs/jquery.waypoints-sticky.min.js',
+            'js/libs/matchMedia.js',
+            'js/libs/matchMedia.addListener.js',
```

Comment:
Ah, yes: IE9 is throwing the error `SCRIPT438: Object doesn't support property or method 'addListener' family-nav.js, line 50 character 13`